api/directions/views.py

```diff
@@ -2116,3 +2118,16 @@ def directions_type_date(request):
     result_direction = list(set(confirm_direction) - set(not_confirm_direction))
 
     return JsonResponse({"results": result_direction})
+
+
+@login_required
+@group_required("Управление иерархией истории")
+def change_owner_direction(request):
+    user = request.user.doctorprofile
+    request_data = json.loads(request.body)
+    new_card_number = request_data.get('new_card_number', None)
```

Comment:
Оно же не может быть пустым
```suggestion
    new_card_number = request_data['new_card_number']
```
и ниже тоже